airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -383,7 +404,11 @@ def trigger_dag_run(
         raise HTTPException(status.HTTP_400_BAD_REQUEST, str(e))
 
 
-@dag_run_router.post("/list", responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]))
+@dag_run_router.post(
+    "/list",
+    responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
+    dependencies=[Depends(requires_access_dag(method="POST", access_entity=DagAccessEntity.RUN))],
```

Comment:
@security.requires_access_dag("GET", DagAccessEntity.RUN)


I don't think we need "post" access for this. It's actually a read.